aip/0194.md

```diff
@@ -0,0 +1,237 @@
+---
+aip:
+  id: 194
+  state: reviewing
+  created: 2019-08-16
+permalink: /194
+redirect_from:
+  - /0194
+---
+
+# Retrying for client libraries
+
+This document aims to lay out guidance about which requests to retry, and for
```

Comment:
This paragraph feels a little odd in an AIP. I'd make it more assertive but introductory. *Something* like:

> RPCs sometimes fail. When one does, the client performing the RPC needs to know whether it is safe to retry the operation. When status codes are used consistently across multiple APIs, clients can respond to failures appropriately.

(Probably more than that...) It might be worth us discussing to what extent this AIP is for clients and to what extent it's for API producers. It seems to be mostly about clients at the moment... @lukesneeringer any concerns around that?